### PR TITLE
FileStructuting

### PR DESCRIPTION
**Pull request by fogapandrew**
Adding New files and structuring the files

### CODE REVIEW

**Review by ivombi — COMMENTED** [review]
General remark about file naming convention. Preferable to use lower case and not sentence case.  E.g Extraction.py -> extraction.py

File: Injestion&PreparationPhase.drawio
Comment (ivombi) [inline Injestion&PreparationPhase.drawio:1]: Convert to jpeg

File: SkillMatchingJobProject [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: Skill-Matching_Job_Porject
+
+conda_env: environment.yml
+
+entry_points:
+  main:
+    command: "python Extraction.py"
```
Comment (ivombi) [inline SkillMatchingJobProject]: Not sure mlflow will run this python file.  Extraction.py is in ETL-System folder. Also avoid '-' in your naming convection

**Review by Njuaka — COMMENTED** [review]
great update

**Review by Njuaka — APPROVED** [review]

**Pull request closed without merging**